raco/algebra.py

```diff
@@ -1525,25 +1537,35 @@ def __hash__(self):
         return ("%s-%s" % (self.opname(), self.relation_key)).__hash__()
 
     def shortStr(self):
-        return "%s(%s)" % (self.opname(), self.relation_key)
+        return "%s(%s, %s)" % (self.opname(),
+                               self.relation_key,
+                               "debroad" if self._debroadcast else "")
```

Comment:
I don't like it either, it's what I inherited from the first version and I can't think of anything better. Note that something like `BroadcastScan` for `debroadcast==False` doesn't really make sense because `debroadcast==False` does not imply that the relation is broadcast. I think a logical `DeBroadcast` operator (which I removed from the first version) would make the visualized query plan clearer, but maybe the next best thing is `DeBroadcast(Scan(<<self. relation_key>>))`?